mock/py/mockbuild/util.py

```diff
@@ -1238,12 +1274,16 @@ def setup_operations_timeout(config_opts):
     _OPS_TIMEOUT = config_opts.get('opstimeout', 0)
 
 @traceLog()
-def do_update_config(log, config_opts, cfg, uidManager, name, skipError=True):
+def do_update_config(log, config_opts, cfg_paths, uidManager):
```

Comment:
why the change of cfg to plural? And similarly in update_config_from_template(). The config is still only one file, isn't it?

Reply:
You ommited name and skipError, but you still use it in load_config()

Reply:
Because here cfg argument is now a list of paths rather then one path:

`do_update_config(log, config_opts, config_opts['config_paths'], uidManager)`